hack/run-in-python-container.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script runs $@ in a python container
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
+cd "${REPO_ROOT}"
+
+PY_IMAGE='python:3.9-slim-buster'
+PYLINTHOME="$(mktemp -d)"
```

Comment:
why is pylint in a tempdir created every run?
does the tempdir need to be bind-mounted at all then?
why don't we delete the tempdir? or use something under /tmp inside the container and don't bind-mount?

Reply:
this was almost copy/pasted from existing script, didn't look into that much detail what `PYLINTHOME` is used for, will take a deeper look

Reply:
It's used by pylint according to https://pylint.pycqa.org/en/latest/faq.html#where-is-the-persistent-data-stored-to-compare-between-successive-runs, for storing pylint analysis results, which is not super critical